ui-components/src/page_components/listing/listing_sidebar/Apply.tsx

```diff
@@ -66,11 +64,26 @@ const Apply = (props: ApplyProps) => {
     return method.type == ApplicationMethodType.FileDownload
   })
 
+  const getDropOffAddress = (addressType: string | undefined) => {
+    return addressType === ListingApplicationDropOffAddressType.leasingAgent
+      ? listing.leasingAgentAddress
+      : addressType === ListingApplicationDropOffAddressType.mailingAddress
+      ? listing.applicationMailingAddress
+      : listing.applicationDropOffAddress
+  }
+
+  const getPickUpAddress = (addressType: string | undefined) => {
```

Comment:
I think we can use `ListingApplicationPickUpAddressType ` as `addressType` type, what do you think?